metadata-ingestion/src/datahub/ingestion/source/salesforce.py

```diff
@@ -620,10 +868,14 @@ def _get_field_description(self, field: dict, customField: dict) -> str:
             prefix = "\\" if text.startswith("#") else ""
             desc += f"\n\n{prefix}{text}"
 
+        if formula:
+            desc += f"\n\nFormula: {formula}"
```

Comment:
if desc is not set but formula is, do we end up with `\n\n` in front of the formula name?

maybe we should do 
```
description_parts: List[str] = []
...

desc = '\n\n'.join(description_parts)
```

Reply:
Updated.